extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -494,7 +494,7 @@ function createDicomWebApi(dicomWebConfig, UserAuthenticationService) {
         const NumberOfFrames = instance.NumberOfFrames;
 
         if (NumberOfFrames > 1) {
-          for (let i = 0; i < NumberOfFrames; i++) {
+          for (let i = 1; i <= NumberOfFrames; i++) {
```

Comment:
If this is frame, call it frame in the for loop.

Reply:
resolved in commit ea05ac7db90672acc420f7c264bf04cd17af885a